src/api/app/views/webui/shared/_input.haml

```diff
@@ -4,4 +4,5 @@
   .col-9.d-flex
     = text_field_tag(filter_item, '',
                      class: 'form-control form-control-sm',
-                     placeholder: placeholder, value: selected_input_value)
+                     placeholder: defined?(placeholder) ? placeholder : '', value: selected_input_value,
+                     type: defined?(type) ? type : 'text')
```

Comment:
```suggestion
                     type: type.presence || 'text')
```
Maybe clearer this way...?

Reply:
Unfortunately no. `type.presence` checks if `type value is nil` but it assumes `type` variable exists. We need to check instead if `type` variable is passed at all: if so, we overwrite it, otherwise it takes the `text` value which is the most common used by default.

Reply:
Ops, true. `type.presence` raises an exception if `type` is not defined. :+1: